modules/safer_mysql/main.tf

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "safer_mysql" {
+  source                          = "../mysql"
+  project_id                      = "${var.project_id}"
+  name                            = "${var.name}"
+  database_version                = "${var.database_version}"
+  region                          = "${var.region}"
+  zone                            = "${var.zone}"
+  tier                            = "${var.tier}"
+  activation_policy               = "${var.activation_policy}"
+  authorized_gae_applications     = "${var.authorized_gae_applications}"
+  disk_autoresize                 = "${var.disk_autoresize}"
+  disk_size                       = "${var.disk_size}"
+  disk_type                       = "${var.disk_type}"
+  pricing_plan                    = "${var.pricing_plan}"
+  maintenance_window_day          = "${var.maintenance_window_day}"
+  maintenance_window_hour         = "${var.maintenance_window_hour}"
+  maintenance_window_update_track = "${var.maintenance_window_update_track}"
+  database_flags                  = "${var.database_flags}"
+  user_labels                     = "${var.user_labels}"
+  backup_configuration            = "${var.backup_configuration}"
+
+  ip_configuration = [
+    {
+      ipv4_enabled = "${var.assign_public_ip}"
+
+      # We never set authorized networks, we need all connections via the
+      # public IP to be mediated by Cloud SQL.
+      authorized_networks = []
```

Comment:
Ehhh, I use `authorized_networks` in my scripts via the `ip_configuration` block. Why hard code this to empty?

Reply:
nvm I see this is just for a new option.

Reply:
Correct, this is just a new opinionated setup with some "safer" defaults, it won't affect anything for the current users.